### ISSUE TITLE
Show site-specific autoplay setting in the website tab address bar menu

### ISSUE THREAD

**Issue by dentistformyeye** [brave/brave-browser#19424]
## Description 

Clicking the icon (usually a lock) on the left of the address bar opens up a menu for editing site-specific settings, but Autoplay doesn't appear in the menu. The only way to enable autoplay on the website tab (as opposed to in settings) is to click the "Autoplay was blocked on this page" icon on the right side of the address bar. After the page refreshes, the only way to disable autoplay again is to go to brave://settings. The autoplay setting should appear in the address bar menu.
(It would also be nice to have a permission lifetime for autoplay. The only options right now are block and always allow.)

## Steps to Reproduce

   1. Disable autoplay in brave://settings/content/autoplay. Delete any entries for youtube.com
   2. Go to https://www.youtube.com/watch?v=5qap5aO4i9A
   3. Click the autoplay-blocked icon on the right side of the address bar and choose "Always allow...". Click done and allow the page to refresh.
![image](https://user-images.githubusercontent.com/51429646/141603104-886304d0-1a69-4508-b238-7701d6927651.png)

   4. Click the lock icon on the left side of the address bar to open the menu.

## Actual result:

No autoplay setting appears in the address bar menu. The icon from step 3 also isn't present.
![image](https://user-images.githubusercontent.com/51429646/141603199-bbf8178c-43e1-4c00-b298-1161b0fbf16c.png)

## Expected result:
The address bar menu includes an autoplay setting.

## Reproduces how often: 

Easily reproduced

## Brave version (brave://version info)

1.31.91 Chromium: 95.0.4638.69 (Official Build) (x86_64)
--
Revision | 6a1600ed572fedecd573b6c2b90a22fe6392a410-refs/branch-heads/4638@{#984}
OS | macOS Version 12.0.1 (Build 21A559)

## Version/Channel Information:

- Can you reproduce this issue with the current release? Yes
- Can you reproduce this issue with the beta channel? Haven't tried
- Can you reproduce this issue with the nightly channel? Haven't tried

## Other Additional Information:

- Does the issue resolve itself when disabling Brave Shields? No
- Does the issue resolve itself when disabling Brave Rewards? No
- Is the issue reproducible on the latest version of Chrome? Haven't tried
